### PR TITLE
64 setup sdk package test

### PR DESCRIPTION
**Pull request by Louai-Zokerburg**
#64

### CODE REVIEW

**Review by 1hachem — CHANGES_REQUESTED** [review]
- after you merge please update the tests for defintions (build instead of push ...)
- add a dockerfile where you run the test, all sample files should be inside docker container

File: Dockerfile
Comment (Louai-Zokerburg) [inline Dockerfile:1]: @1hachem Need assistance here to setup tests environment for tests, what really confused me is in of the tests tests/test_defenitions.py::test_build i need to read an actual Dockerfile and build it, how can i do so in the container?!

Comment (1hachem) [reply Dockerfile:1]: lets avoid running docker (subprocess) inside docker (test container) inside docker (github action docker)...

Comment (1hachem) [reply Dockerfile:1]: [checkout this](https://jpetazzo.github.io/2015/09/03/do-not-use-docker-in-docker-for-ci/) maybe interesting for you

**Review by strlrd-29 — APPROVED** [review]

**Pull request merged**